gigascience/gigadb-website
# Sort dataset files by size

# Pull request for issue: #2067

- sort header sorts files by size rather than a-z in dataset view pages, eg http://gigadb.gigasciencejournal.com/dataset/100035

## How to test?

- navigate to http://gigadb.gigasciencejournal.com/dataset/100035
- click on size header
- check that rows are sorted by actual size not a-z

## How have functionalities been implemented?

customized the sorting function for the size column

## Any issues with implementation?

--

## Any changes to automated tests?

--
